worker/httpserver/worker_test.go

```diff
@@ -243,7 +243,7 @@ func (s *WorkerSuite) TestMinTLSVersion(c *gc.C) {
 	tlsConfig.MaxVersion = tls.VersionSSL30
 
 	conn, err := tls.Dial("tcp", parsed.Host, tlsConfig)
-	c.Assert(err, gc.ErrorMatches, ".*protocol version not supported")
+	c.Assert(err, gc.ErrorMatches, "tls: no supported versions satisfy MinVersion and MaxVersion")
```

Comment:
I assume the reason we can actually change this is hidden in the testing package change.

Reply:
I don't see how this would work with go-1.11 since it gives "tls: protocol version not supported" there.
This looks more like you tested using go-1.12 on the 2.6 codebase.

Reply:
I tested both go-1.11 and go-1.12.

Reply:
I've tweaked the error message to ensure we do handle 1.11 and 1.12